change/@fluentui-react-portal-3d88ca5d-d80e-40b2-9cb7-8bb39a62da74.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
```

This also updates only stories, so should be `none`

```suggestion
  "type": "none",
```